bundles/ch.elexis.core.ui.contacts/src/ch/elexis/core/ui/contacts/views/Patientenblatt2.java

```diff
@@ -790,7 +790,7 @@ public void refreshUi(){
 			+ StringTool.unNull(actPatient.getVorname()) + " (" //$NON-NLS-1$
 			+ actPatient.getPatCode() + ")"); //$NON-NLS-1$
 		inpAdresse.setText(actPatient.getPostAnschrift(false), false, false);
-		UserSettings.setExpandedState(ecZA, "Patientenblatt/Zusatzadressen"); //$NON-NLS-1$
+		UserSettings.setExpandedState(ecZA, Messages.Patientenblatt2_additionalAdresses); //$NON-NLS-1$
```

Comment:
This is not correct. It refers to field, not a message.
But it made me look around I found various problems which AdressTypes and RelationShipTypes.
Look at my commit

Reply:
I understand. Going to close this pull request.